mantidimaging/gui/windows/spectrum_viewer/model.py

```diff
@@ -99,6 +99,7 @@ def __init__(self, presenter: SpectrumViewerWindowPresenter):
         self._roi_id_counter = 0
         self._roi_ranges = {}
         self.special_roi_list = [ROI_ALL]
+        self._rois: list[SensibleROI] = []
```

Comment:
I think this is left over from something else. We don't want to add more state to the model.